AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/CampaignAdminControllerTests.cs

```diff
@@ -82,29 +88,63 @@ public void IndexReturnsCorrectDataWhenUserIsOrgAdmin()
             controller.SetClaims(claims);
 
             ViewResult view = (ViewResult)controller.Index();
-
-            // verify the fetch was called
             mockMediator.Verify(mock => mock.Send(It.Is<CampaignListQuery>(c => c.OrganizationId == OrganizationId)));
 
             // Org admin should only see own campaigns
             IEnumerable<CampaignSummaryModel> viewModel = (IEnumerable<CampaignSummaryModel>)view.ViewData.Model;
             Assert.NotNull(viewModel);
             Assert.Equal(viewModel.Count(), 1);
             Assert.Equal(viewModel.First().OrganizationId, OrganizationId);
-            
         }
 
-
-        [Fact(Skip = "NotImplemented")]
-        public void IndexReturnsCorrectViewModel()
+        public void IndexReturnsCorrectDataWhenUserIsNotOrgAdmin()
         {
+            int OrganizationId = 99;
+            var mockMediator = new Mock<IMediator>();
+            mockMediator.Setup(x => x.Send(It.Is<CampaignListQuery>(c => c.OrganizationId == null)))
+                .Returns((CampaignListQuery q) => {
+                    // return some models 
+                    List<CampaignSummaryModel> ret = new List<CampaignSummaryModel>();
+                    ret.Add(new CampaignSummaryModel { OrganizationId = OrganizationId });
+                    ret.Add(new CampaignSummaryModel { OrganizationId = OrganizationId + 1 });
+                    return ret;
+                }
+            );
+            var mockImageService = new Mock<IImageService>();
+            CampaignController controller = new CampaignController(mockMediator.Object, mockImageService.Object);
+
+            List<Claim> claims = new List<Claim>
+            {
+                new Claim(AllReady.Security.ClaimTypes.UserType, UserType.SiteAdmin.ToString()),
+            };
+            controller.SetClaims(claims);
+
+            // All campaigns returned when not OrgAdmin
+            ViewResult view = (ViewResult)controller.Index();
+
+            // verify the fetch was called
+            mockMediator.Verify(mock => mock.Send(It.Is<CampaignListQuery>(c => c.OrganizationId == null)));
```

Comment:
I think this is good as far as unit tests go. In fact, I would argue that for the controller that if the ViewModel is of the expected type and you've invoked the bus with the correct parameters, then the other assertions (around count) are superfluous. ...and you may _just_ be testing your test.